apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/DownloadObjectOptions.cs

```diff
@@ -55,12 +56,18 @@ public sealed class DownloadObjectOptions
         /// </summary>
         public long? IfMetagenerationNotMatch { get; set; }
 
+        /// <summary>
+        /// If non-null, limits the range of bytes of the object to download.
+        /// </summary>
+        public RangeHeaderValue Range { get; set; }
+
         internal void ModifyDownloader(MediaDownloader downloader)
         {
             if (ChunkSize != null)
             {
                 downloader.ChunkSize = ChunkSize.Value;
             }
+            downloader.Range = Range;
```

Comment:
Should this only be set if Range is not null? For something like this I guess the risk is pretty low that you'd be unintentionally overwriting something.